components/openindiana/time-slider/Makefile

```diff
@@ -21,7 +21,7 @@ COMPONENT_VERSION=	0.2.101
 COMPONENT_LICENSE=	CDDL
 COMPONENT_LICENSE_FILE=	$(COMPONENT_NAME).license
 
-COMPONENT_REVISION=$(shell cd $(COMPONENT_SRC); git rev-list HEAD --count)
+COMPONENT_REVISION=$(shell cd $(COMPONENT_SRC); git rev-list HEAD --count).1
```

Comment:
This will get us time-slider@0.2.101-2017.0.0.107.1 version instead of time-slider@0.2.101-2017.0.0.107. Seems a bit strange. As for rebuild, I'll just do gmake clean on build server. And we don't want rebuild for this change, just republishing. 
Sorry, we don't do such things for components like illumos-gate, gfx-drm and so on. If it was really important, we could bump UPDATENUM , but it's just a trifle.